src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import "./styles.scss";
+import { IDropdownProps } from "../../types/interfaces";
+import { ReactComponent as Arrow } from "../../assets/arrowDown.svg";
+export default function Dropdown({ items, selected }: IDropdownProps) {
+  const [isOpen, setIsOpen] = useState(false);
+  const handleDropdown = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+    setIsOpen(!isOpen);
+  };
+  return (
+    <div className={"container"}>
+      <button className="dropdown" onClick={handleDropdown}>
+        {selected}
+        <Arrow className={"arrow " + (isOpen ? "down" : "")} />
```

Comment:
```suggestion
        <Arrow className={`arrow ${isOpen ? "down" : ""}`} />
```

Reply:
What is the reason for doing it this way?